src/test/java/GameSearchTest.java

```diff
@@ -0,0 +1,68 @@
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.BasePage;
+import pages.HomePage;
+import pages.SearchPage;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.Duration;
+
+@Test
+public class GameSearchTest {
+
+    public void gameSearchTest() {
+        // initialisation
+
+        DesiredCapabilities desiredCapabilities = new DesiredCapabilities();
+        desiredCapabilities.setBrowserName("Chrome");
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--incognito");
+        desiredCapabilities.setCapability(ChromeOptions.CAPABILITY, options);
+
+        WebDriver browser = new ChromeDriver(options);
+        WebDriverWait wait = new WebDriverWait(browser, Duration.ofMillis(1000));
+
+        browser.manage().window().maximize();
+        String url = "http://store.steampowered.com/";
+        String keyword = "Dota 2";
+
+        //Working on Home page
+        HomePage homePage = new HomePage(browser);
+        // Go to the main page
+        homePage.browseMainPage(url);
```

Comment:
The utility class should be responsible for following links and working with the browser